docs/advanced/input_files/input-main.md

```diff
@@ -913,7 +913,6 @@ These variables are used to control the numerical atomic orbitals related parame
 - **Description**: choose the number of streams in GPU when we compute the `LCAO`. According to different devices , we may have different effects.For most devices,the stream is
 enough when the number is bigger then 2.
 - **Default** : "4" 
-[back to top](#full-list-of-input-keywords)
```

Comment:
why this line was deleted?

Reply:
Originally thought this was unnecessary, but later realized this is a format for markdown documents, so I added it back